doc/license.md

```diff
@@ -77,8 +77,7 @@ should be copied to `./plugins.cfg` before calling `./configure-plugins` to take
 
 ### Less restrictive than LGPL
 
-* libr/asm/arch/gnu: GPLv2
```

Comment:
Dont remove this line

Reply:
The code in this subdirectory is from gnu binutils

Reply:
This subdirectory does not even exist.

Reply:
Correct. Its libr/asm/arch/*/gnu